.github/cspell.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": "0.2",
+  "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json",
+  "dictionaries": [
+    "vgv_allowed",
+    "vgv_forbidden"
+  ],
+  "dictionaryDefinitions": [
+    {
+      "name": "vgv_allowed",
+      "path": "https://raw.githubusercontent.com/verygoodopensource/very_good_dictionaries/main/allowed.txt",
+      "description": "Allowed VGV Spellings"
+    },
+    {
+      "name": "vgv_forbidden",
+      "path": "https://raw.githubusercontent.com/verygoodopensource/very_good_dictionaries/main/forbidden.txt",
+      "description": "Forbidden VGV Spellings"
+    }
+  ],
+  "useGitignore": true,
+  "words": [
+    "Mockingjay",
+    "korzonkiee",
+    "Jignesh",
+    "Fullscreen",
+    "allisonryan"
```

Comment:
where are some of these github usernames mentioned?

Reply:
In the changelog